src/control/lib/netdetect/netdetect.go

```diff
@@ -1194,10 +1199,11 @@ func ScanFabric(provider string, excludes ...string) ([]FabricScan, error) {
 				case allHFIUsed:
 					for deviceID := 0; deviceID < hfiDeviceCount; deviceID++ {
 						deviceScanCfg.targetDevice = fmt.Sprintf("hfi1_%d", deviceID)
-						resultsMap, ScanResults, err = createFabricScanEntry(deviceScanCfg, C.GoString(fi.fabric_attr.prov_name), devCount, resultsMap, ScanResults, excludeMap)
+						scanResults, err := createFabricScanEntry(deviceScanCfg, C.GoString(fi.fabric_attr.prov_name), devCount, resultsMap, excludeMap)
```

Comment:
having slightly differently-capitalized variable names in the same function seems destined to cause confusion. Maybe devScanResults instead?

Reply:
Heh.  I guess that could go either way.  I've changed it though.